package.json

```diff
@@ -29,7 +29,7 @@
         "@eslint/js": "^8.43.0",
         "@types/eslint": "^8.44.6",
         "@typescript-eslint/parser": "^5.60.0",
-        "esbuild": "^0.18.7",
+        "esbuild": "^0.19.0",
```

Comment:
We should probably make this `~0.19.0` as the breaking changes happen in the 'minor' version number https://github.com/evanw/esbuild/releases/tag/v0.19.0
```suggestion
        "esbuild": "~0.19.0",
```

Reply:
esbuild was only used to compile `import.meta`, and will be no longer used once https://github.com/eslint-community/eslint-plugin-n/pull/139 get landed. so, it's not really an issue to me. 😄

Reply:
@scagood For any `0.x.y` version the `^` and `~` works identically, so no need to change :) A good place to test oit ranges is: https://semver.npmjs.com/ 🙂

Reply:
Ooh! That's cool!